content/index.pl.md

```diff
@@ -0,0 +1,252 @@
+lang: pl   # Change "en" to the translated-language shortcode
+last_updated: 2024-05-31   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+translators:
+  - name: "Stefan Wajda"
+ 
+contributors:
+  - name: "Michał Sobkowiak"   # Replace Jan Doe with contributor name, or delete this line if none
+
+
+github:
+    repository: w3c/wai-intro-accessibility
+    branch: gh-pages
+    path: content/index.pl.md   # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+
+permalink: /fundamentals/accessibility-intro/pl   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
+ref: /fundamentals/accessibility-intro/     # Do not change this
+
+changelog: /fundamentals/accessibility-intro/changelog/ # Do not change this
+layout: default
+  
+# In the footer below:
+# Do not change the dates
+# Do not translate CHANGELOG
+# Translate the other words, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: >
+  <p><strong>Data:</strong> Aktualizacja 20 listopada 2023. Pierwsza publikacja w lutym 2005. DZIENNIK ZMIAN.</p>
+  <p><strong>Redaktorka:</strong> <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Opracowane przez Grupę Roboczą ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>).</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Podsumowanie" class="full" %}
+{:/}
+
+Gdy strony i narzędzia internetowe są odpowiednio zaprojektowane i napisane, mogą z nich korzystać osoby z niepełnosprawnościami. Obecnie jednak wiele witryn i narzędzi jest tworzonych z barierami dostępności, które utrudniają lub uniemożliwiają korzystanie z nich niektórym osobom.
+
+Zwiększenie dostępności przynosi korzyści jednostkom, firmom i społeczeństwu.  Międzynarodowe standardy internetowe określają, co jest potrzebne do zapewnienia dostępności.
+
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Treść strony" class="full" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+<span class="box-h box-h-simple box-h-full">Materiały pokrewne</span><br>
+{% include video-link.html title="Film wprowadzający do dostępności internetowej i standardów W3C <em>(4 minuty)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits/" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Spojrzenie na dostępność {#context}
+
+<blockquote class="pull">
+  <p>Siła Internetu tkwi w jego uniwersalności.<br />
+    Dostęp dla każdego bez względu na niepełnosprawność jest jego zasadniczym atrybutem.</p>
```

Comment:
Original quote reads "Access by everone regardless of disability is _**an**_ essential aspect".

It seems the translation reads "is **_its_** essential attribute"

@zwiastunsw Would you adapt the translation?

Reply:
```suggestion
    Dostęp dla każdego bez względu na niepełnosprawność jest jego zasadniczym aspektem.</p>
```